book/src/specs/kimchi.md

```diff
@@ -1196,6 +1196,90 @@ Given value `v2` the layout looks like this
 where the notation `v2ci` and `v2pi` defined in the "Layout" section above.
 
 
+#### Foreign Field Addition
+
+These circuit gates are used to constrain that
+
+    $$left_input +/- right_input = field_overflow * foreign_modulus + result$$
+
+ Documentation:
+
+  For more details please see the [FFadd RFC](../rfcs/ffadd.md)
```

Comment:
I cannot find this file on this branch.  Is it missing from the commit?

Reply:
added again after our chat